utils/meta-cli/test/test.js

```diff
@@ -0,0 +1,50 @@
+import child_process from 'child_process';
+
+const exec = async (cmd) => {
+  let result = new Promise((resolve, reject) => {
+    child_process.exec(`node src/app.js ${cmd}`, (error, stdout) => {
+      if (error) {
+        reject(error);
+      }
+      resolve(stdout);
+    });
+  });
+  // console.log(await result);
+  return await result;
+};
+
+describe('Test gear-meta', () => {
+  test('--help', async () => {
+    expect(await exec('--help')).toBeDefined();
+  });
+
+  test('--version', async () => {
+    expect(await exec('--version')).toBeDefined();
+  });
+
+  test('decode PING', async () => {
+    expect(await exec('decode 0x50494e47 -t String')).toBeDefined();
+  });
+
+  test('encode PING', async () => {
+    expect(await exec('encode PING -t String')).toBeDefined();
+  });
+
+  test('encode init_input', async () => {
+    expect(
+      await exec(`encode '{"amount": 8, "currency": "GRT"}' -t init_input -m test/wasm/demo_meta.meta.wasm`),
```

Comment:
Could you add a more meaningful check on the result instead of `toBeDefined`? Since we are not sure if the result is correct or not when introducing modifications.

Reply:
This tool use @gear-js/api to encode and decode payloads and for everything else. So there is no need to check right results here, because it is checked on @gear-js/api. And this lib can't affect it

Reply:
I check here only commands execution

Reply:
> This tool use @gear-js/api to encode and decode payloads and for everything else. So there is no need to check right results here, because it is checked on @gear-js/api. And this lib can't affect it

Understand. Thanks.